python_modules/dagster-test/dagster_test/graph_job_op_toys/schedules.py

```diff
@@ -57,160 +29,85 @@ def _toys_tz_info():
+        return {}
 
-    return partition_set.create_schedule_definition(
-        schedule_name=schedule_name,
-        cron_schedule="* * * * *",  # tick every minute
-        partition_selector=backfilling_partition_selector,
-        should_execute=_should_execute,
-        execution_timezone=_toys_tz_info(),
-    )
+    unreliable_weekly_job = unreliable.to_job(config=unreliable_config)
+    unreliable_weekly_schedule = schedule_from_partitions(unreliable_weekly_job)
 
+    return unreliable_weekly_schedule
 
-def materialization_schedule():
-    # create weekly partition set
-    schedule_name = "many_events_partitioned"
-    partition_set = PartitionSetDefinition(
-        name="many_events_minutely",
-        pipeline_name="many_events",
-        partition_fn=date_partition_range(start=datetime.datetime(2020, 1, 1)),
-        run_config_fn_for_partition=lambda _: {"intermediate_storage": {"filesystem": {}}},
-    )
 
-    def _should_execute(context):
-        return backfill_should_execute(context, partition_set)
+def hourly_materialization_schedule():
+    @hourly_partitioned_config(start_date=datetime.datetime(2021, 1, 1), timezone=_toys_tz_info())
+    def hourly_materialization_config(_start, _end):
+        return {}
 
-    return partition_set.create_schedule_definition(
-        schedule_name=schedule_name,
-        cron_schedule="* * * * *",  # tick every minute
-        partition_selector=backfilling_partition_selector,
-        should_execute=_should_execute,
-        execution_timezone=_toys_tz_info(),
+    return schedule_from_partitions(
+        many_events.to_job("many_events_hourly_config", config=hourly_materialization_config),
+        name="many_events_hourly",
     )
 
 
-@hourly_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
-def hourly_materialization_schedule():
-    return {}
-
-
-@daily_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
 def daily_materialization_schedule():
-    return {}
+    @daily_partitioned_config(start_date=datetime.datetime(2021, 1, 1), timezone=_toys_tz_info())
+    def daily_materialization_config(_start, _end):
+        return {}
+
+    return schedule_from_partitions(
+        many_events.to_job("many_events_daily_config", config=daily_materialization_config),
+        name="many_events_daily",
+    )
 
 
-@weekly_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
 def weekly_materialization_schedule():
-    return {}
+    @weekly_partitioned_config(start_date=datetime.datetime(2021, 1, 1), timezone=_toys_tz_info())
+    def weekly_materialization_config(_start, _end):
+        return {}
+
+    return schedule_from_partitions(
+        many_events.to_job("many_events_weekly_config", config=weekly_materialization_config),
```

Comment:
I think it would make more sense to name the job "many_events_weekly" than "many_events_weekly_config".  Then you could omit the name argument to `schedule_from_partitions` and end up with a default sensible name for the schedule "many_events_weekly_schedule".